apps/web/next.config.js

```diff
@@ -65,6 +65,9 @@ plugins.push(withTM);
 
 /** @type {import("next").NextConfig} */
 const nextConfig = {
+  images: {
+    domains: [`${new URL(process.env.NEXT_PUBLIC_WEBSITE_URL).hostname}`, "www.gravatar.com"],
+  },
```

Comment:
We're not using ´Image´ with gravatars anymore so this is not being used.

Reply:
Leaving the first value as it comes handy for other `<Image>` usages with local images

Reply:
It's not needed as is the same domain where the website is being served. It is only needed for external domains.